.github/workflows/run_instrumentation_tests.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+CI_AGP_VERSION=$1
+
+function printthreads {
+  echo "Thread dump"
+  jps|grep -E 'KotlinCompileDaemon|GradleDaemon'| awk '{print $1}'| xargs jstack
+}
+
+trap printthreads SIGINT
+
+echo "Running with Proguard!"
+timeout --signal=SIGINT 5m ./gradlew connectedExternalStagingAndroidTest -Pandroid.enableR8=false --stacktrace -PkeeperTest.agpVersion=${CI_AGP_VERSION}
```

Comment:
Any reason to have a timeout for proguard but not r8?

Reply:
Leftover from when I was seeing hangups on CI, let me just remove it all together